src/wallet/rpcdump.cpp

```diff
@@ -663,51 +662,40 @@ UniValue importwallet(const JSONRPCRequest& request)
             CPubKey pubkey = key.GetPubKey();
             assert(key.VerifyPubKey(pubkey));
             CKeyID keyid = pubkey.GetID();
-            if (pwallet->HaveKey(keyid)) {
-                pwallet->WalletLogPrintf("Skipping import of %s (key already present)\n", EncodeDestination(PKHash(keyid)));
+
+            std::map<CKeyID, CKey> privkey_map;
+            privkey_map.emplace(keyid, key);
+
+            if (!pwallet->ImportPrivKeys(privkey_map, time)) {
```

Comment:
In commit "Have importwallet use ImportPrivKeys and ImportScripts" (991ecfdbbb289bd6e4c50fc63f9201edbca14f70)

IMO, would be clearer to call `ImportPrivKeys({keyid, key}}, time)` than have the extra map variable.

Reply:
Done